input/pagecontent/RAD-Y1.md

```diff
@@ -103,40 +103,32 @@ There is a common set of attributes included in radiology diagnostic reports. Ta
-| Examination      | DiagnosticReport.code | See [IMR DiagnosticReport](StructureDefinition-imr-diagnosticreport.html) for details | Code for the type of diagnostic report, may be the same as the study procedure code |
-| Indication       | DiagnosticReport.extension[indication] | See [IMR DiagnosticReport](StructureDefinition-imr-diagnosticreport.html) for details | Each value can be either a string or a CodeableConcept |
-| Technique        | DiagnosticReport.result.method -> IMRObservation.method | Technique details is set in the same observation  resource that captures the finding. i.e. IMRObservation.code = LOINC#59776-5 "Procedure Findings" <br><br> See [IMR DiagnosticReport](StructureDefinition-imr-diagnosticreport.html) and [IMR Observation](StructureDefinition-imr-observation.html) for details | |
-| Comparison       | DiagnosticReport.extension[comparisonStudy] | Can be either an IMRImagingStudy or IMRDiagnosticReport <br><br> See [IMR DiagnosticReport](StructureDefinition-imr-diagnosticreport.html) for details | |
-| Report Section   | DiagnosticReport.result.valueString -> IMRObservation.valueString | Identified by IMRObservation.code. See [IMR DiagnosticReport](StructureDefinition-imr-diagnosticreport.html) and [IMR Observation](StructureDefinition-imr-observation.html) for details | The code is used to identify what *section* the observation belongs to. For example, LOINC code __59776-5__ represents a procedure finding and LOINC code __19005-8__ represents a narrative impression. <br><br> Highly recommended to encode a single finding or impression per IMR Observation, but permitted to encode all findings as a single string and all impressions as a single string to bridge existing applications. <br><br> Also See Note 2 |
-{: .grid}
-
-> Note 1: There is no IMR-defined FHIR resource profile for the resource. An implementation may use other FHIR resource profiles applicable for their deployment.
-
-> Note 2: In common cases, there is no direct association between findings and impressions except that they are associated with the same DiagnosticReport Resource. 
-
-In addition to the common set above, there are a number of useful optional attributes that the Sender may include, if applicable.
-
-**Table 2:4.Y1.4.1.2.2.1-2: Useful Optional Attributes in Radiology Diagnostic Report**
-
 | Report Attribute | FHIR Resource Mapping | Cardinality | Additional Constraint | Note |
 |------------------|-----------------------|-------------|-----------------------|------|
+| Organization     | DiagnosticReport.performer | 1..* | See [IMR DiagnosticReport](StructureDefinition-imr-diagnosticreport.html) for details | The organization which is responsible for the diagnostic report. <br><br> See Note 1 |
```

Comment:
I didn't combine the two tables because these two tables are not complete in the sense of what are the required attribute and what are the complete list of optional attributes. There are more defined in the actual artifacts page. What I want to highlight in these tables are the common set of attributes that the user would expect. In the actual resources, there are a few more required attributes (e.g. `category` that is more an internal indication of report type). Also there are many more optional attributes that are part of the resource but less relevant for rad diagnostic report, and I omitted them below.

Thought?

Reply:
Yes.   This is a mapping table.  It does not specify all of the attributes that must be sent.  That is the main reason to remove the cardinality column.   If this is all the mapping that RAD-Y1 defines, the tables can be combined.   If you are implying that the second table is "optional" or "recommended mapping", then keeping them separate is good.